core/src/main/scala/cats/data/Continuation.scala

```diff
@@ -0,0 +1,108 @@
+package cats
+package data
+
+import java.io.Serializable
+import scala.annotation.tailrec
+
+sealed abstract class Continuation[O, +I] extends Serializable {
```

Comment:
`extends Product with Serializable`

Also Scaladoc saying `Continuation[O, I]` is `(I => O) => O`. Seeing the type parameters order reversed confused me for a second before I realized it was probably for the `Monad` instance